frontend/v1/app/pages/api/orders.ts

```diff
@@ -0,0 +1,72 @@
+
+import type { NextApiRequest, NextApiResponse } from 'next'
+
+type Order = {
+  firstName: string,
+  lastName: string,
+  destination: {
+    address: string,
+    city: string,
+    zipcode: string,
+  }
+  lines: Array<{
+    product: string,
+    quantity: number,
+  }>
+}
+
+let orders : Array<Order> = [];
+
+const handler = (req: NextApiRequest, res: NextApiResponse<Order|string>) => {
+  if (req.method === 'POST') {
+    const data = req.body as Order
+    if (data.firstName === ("" || null || undefined)) {
```

Comment:
nit: this is usually how we test it (see [here](https://developer.mozilla.org/en-US/docs/Glossary/Falsy))
(same for other ifs in this file)
```suggestion
    if (!data.firstName) {
```